1209/9317/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Palawan-Tx
+owner: Kosagi
+license: CC4.0-BY-SA (hardware) / MUT (firmware)
```

Comment:
Er, yes, that's a typo.  Would you fix it, or should I fix it and submit another PR?